doc/admin/realm_config.rst

```diff
@@ -193,6 +193,54 @@ using the **kdc**, **master_kdc**, **admin_server**, and
 explicit server locations, providing SRV records will still benefit
 unconfigured clients, and be useful for other sites.
 
+.. _kdc_discovery:
+
+KDC Discovery
+-------------
+
+As of MIT krb5 1.15, clients can also locate KDCs in DNS through URI
+records (:rfc:`7553`).  Limitations with the SRV record format may
+result in extra DNS queries in situations where a client must failover
+to other transport types, or find a master server.  The URI record can
+convey more information about a realm's KDCs with a single query.
+
+The client performs a query for the following URI records:
+
+* ``_kerberos.REALM`` for fiding KDCs.
+* ``_kerberos-adm.REALM`` for finding kadmin services.
+* ``_kpasswd.REALM`` for finding password services.
+
+The URI record includes a priority, weight, and a URI string that
+consists of case-insensitive colon separated fields:
+
+``scheme:[flags]:transport:residual``
```

Comment:
This is currently formatted a paragraph, which doesn't make sense.  It could be formatted as an example (using ""::" above and indenting it by four spaces), but I would instead bring it up into the previous paragraph, saying "... separated fields, in the form ``...``".